lib/taurus/core/taurusmanager.py

```diff
@@ -120,15 +125,28 @@ def addJob(self, job, callback=None, *args, **kw):
         :param args: (list) list of arguments passed to the job
         :param kw: (dict) keyword arguments passed to the job
         """
-        if self._serialization_mode == TaurusSerializationMode.Concurrent:
+        if kw.has_key("taurus_serialization_mode"):
+            serialization_mode = kw.pop("taurus_serialization_mode")
+        else:
+            serialization_mode = self._serialization_mode
+
+        if serialization_mode == TaurusSerializationMode.Concurrent:
             if not hasattr(self, "_thread_pool") or self._thread_pool is None:
                 self.info("Job cannot be processed.")
                 self.debug(
-                    "The requested job cannot be processed. Make sure this manager is initialized")
+                    "The requested job cannot be processed. " +
+                    "Make sure this manager is initialized")
                 return
             self._thread_pool.add(job, callback, *args, **kw)
         else:
-            job(*args, **kw)
+            if not hasattr(self, "_sthread_pool") or \
```

Comment:
PEP8. Avoid using `\`. In fact, since you do not need the check on hasattr, you do not need to break the line